dwr-hecdssvue/build.gradle

```diff
@@ -0,0 +1,81 @@
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+configurations {
+    eclipseLibs
+    implementation.extendsFrom(eclipseLibs)
+}
+
+dependencies {
+    //additional dependencies
+    implementation libs.hec.monolith
+    implementation libs.hec.nucleus.metadata
+    implementation libs.dssgui
+
+    //module dependencies
+    implementation project(':third-party')
+    implementation project(':wrims-ide')
+
+    //eclipse libs
+    implementation fileTree(dir: '../eclipse-luna-libs', include: ['*.jar'])
+
+    //add test dependency on junit
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
```

Comment:
Version catalog

Reply:
Test impelmentation has been removed from the build.gradle file.